src/main/java/uk/gov/pay/api/agreement/resource/AgreementsApiResource.java

```diff
@@ -86,7 +86,7 @@ public AgreementsApiResource(AgreementService agreementService,
                     @ApiResponse(responseCode = "401",
                             description = RESPONSE_401_DESCRIPTION),
                     @ApiResponse(responseCode = "422",
-                            description = RESPONSE_422_DESCRIPTION,
+                            description = "Your request failed. Check the `code` and `description` in the response to find out why your request failed.",
```

Comment:
I wonder whether this is a better description to be in the `RESPONSE_422_DESCRIPTION` constant to be used as the default, as it applies to all endpoints